doc/test-framework/writing-e2e-tests.md

```diff
@@ -250,6 +252,16 @@ $ kubectl create namespace operator-test
 $ operator-sdk test local ./test/e2e --namespace operator-test
 ```
 
+To run the operator itself locally during the tests instead of starting a deployment in the cluster, you can use the
```

Comment:
How about adding a sub section heading for `--up-local`? 
It's becoming a little hard to read all the different configurations/modes we can run the `test local` command at a glance.